packages/@aws-cdk/aws-appconfig-alpha/lib/configuration.ts

```diff
@@ -302,6 +302,24 @@ abstract class ConfigurationBase extends Construct implements IConfiguration, IE
     this.extensible.addExtension(extension);
   }
 
+  /**
+   * Deploys the configuration to the specified environment.
+   *
+   * @param environment The environment to deploy the configuration to
+   */
+  public deploy(environment: IEnvironment) {
```

Comment:
```suggestion
  public deployTo(environment: IEnvironment) {
```
I think we should use this function to create the deployment in [deployConfigToEnvironments](https://github.com/aws/aws-cdk/blob/78b34accfa1cba88cc412b04df42ea5819c2cf4c/packages/%40aws-cdk/aws-appconfig-alpha/lib/configuration.ts#L322-L331) as well to avoid code repetitions.

Reply:
Hm yeah I had this change, but it wouldn't allow this naming convention because the `deployTo` object already exists

Reply:
Didn't see the edit on the comment above. `deployConfigToEnvironments` has different behavior and shouldn't be used here. I can however create another method to reference to avoid the duplicate code.